.github/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+## Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context, including:
+
+- What does this PR do?
+- Why are these changes needed?
+- How were these changes implemented and what do they affect?
+
+## Related Issues:
+
+Use [Github semantic linking](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) to address any open issues this PR relates to or closes.
+
+- Fixes # (issue number, if applicable)
+
+- Closes # (issue number, if applicable)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] New Release
+- [ ] Documention
+
+## Checklist:
+
+- [ ] My change requires a change to the documentation.
+- [ ] I have updated the documentation accordingly (if applicable).
+- [ ] My change requires storage migration.
+- [ ] I have included the required migration (if applicable).
```

Comment:
this is critical to me to add if we follow such check list idea:
 - [ ] I have tested the required migration (if applicable).

Reply:
Thanks. added.